examples/CustomDataframe/frontend/src/streamlit/streamlit.ts

```diff
@@ -115,10 +115,10 @@ export class Streamlit {
    * value = st.my_component(...)
    * st.write(value) # -> "ahoy!"
    *
-   * The value must be serializable into JSON.
+   * The value must be serializable into JSON or pandas.DataFrame.
```

Comment:
pandas.DataFrame doesn't exist in javascript so this is a bit confusing. What about:

"The value must be serializable into JSON or ArrowTable" (or whatever version of that statement is most correct)